returnn/torch/engine.py

```diff
@@ -506,6 +516,10 @@ def train_epoch(self):
                     batch_size_info=_get_batch_size_info(extern_data) if self._log_batch_size else None,
                     log_memory_usage_device=self._device if self._log_memory_usage else None,
                 )
+                if self._tensorboard_writer:
+                    # write losses/errors to tensorboard
+                    for key, val in eval_info.items():
+                        self._tensorboard_writer.add_scalar(f"train/{key}", val, global_step=self.global_train_step)
```

Comment:
There was some discussion on the needed disk space (and maybe also IO is an issue). It can become quite large.

Maybe we should do this only for every N steps? So some extra check `and self.global_train_step % N == 0` or so. And `N` maybe configurable?

Reply:
yes. should we then have `use_tensorboard` and an additional `tensorboard_opts` in the config?

Reply:
Yes. But there maybe should also be a reasonable default for N? E.g. 100 or so? Or 1000?